integration/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM golang:1.14-alpine AS build_base
+RUN apk add --no-cache git
+WORKDIR /build
+RUN go get -d -v github.com/kr/pretty
+RUN go install -v github.com/kr/pretty
+COPY . .
+RUN go test -c -o gadget-integration.test ./...
+RUN pwd;ls -l
```

Comment:
Is there any reason for these commands?

Reply:
It was for debugging, I forgot to remove it.